torch/csrc/autograd/python_variable.cpp

```diff
@@ -159,6 +159,19 @@ static PyObject* THPVariable_make_subclass(PyObject* _ignored, PyObject* args, P
 typedef PyObject *(*getter)(PyObject *, void *);
 typedef int (*setter)(PyObject *, PyObject *, void *);
 
+PyObject *THPVariable_get_T(THPVariable *self)
+{
+  HANDLE_TH_ERRORS
+  auto& var = self->cdata;
+  int64_t n = var.dim();
+  DimVector transpose_dims;
+  for (int64_t i = n - 1; i >= 0; --i) {
+    transpose_dims.push_back(i);
+  }
+  return THPVariable_Wrap(var.permute(transpose_dims));
```

Comment:
Does this work for scalars, like `torch.tensor(5)`?

Reply:
@fmassa Yes it works fine. It is a no-op, same as Numpy.

```python
>>> t=torch.tensor(5)
>>> t.T
tensor(5)
>>> a=np.array(5)
>>> a.T
array(5)
```

Reply:
I will add a test for this case.